app/javascript/components/rooms/room/Room.jsx

```diff
@@ -19,7 +22,9 @@ export default function Room() {
   const { friendlyId } = useParams();
   const { isLoading, data: room } = useRoom(friendlyId);
   const startMeeting = useStartMeeting(friendlyId);
+  const { isLoadingRunning, data: isRunning } = useMeetingRunning(friendlyId);
```

Comment:
I suspect that `isLoadingRunning` here is `undefined`?
To rename it I suggest the use of the following syntax `const {isLoading: isLoadingRunning, data: isRunning} = useMeetingRunning(...);`

Reply:
Or maybe avoid destructing the returned value and use namespaces `const meetingRunningQuery = useMeetingRunning(...);`